crate/test/jepsen/crate_test.clj

```diff
@@ -58,7 +58,7 @@
                                   #(jepsen/run!
                                      (dirty-read/test
                                        {:es-ops      es-ops
-                                        :crate-version "2.1.8-1~jessie_all"
+                                        :tarball "https://cdn.crate.io/downloads/releases/nightly/crate-admin-latest.tar.gz"
```

Comment:
This is the admin-ui url, it should be: https://cdn.crate.io/downloads/releases/nightly/crate-latest.tar.gz